web/src/app/page.tsx

```diff
@@ -0,0 +1,116 @@
+import Image from 'next/image'
+
+export default function Home() {
+  return (
+    <main>
+      <h1>Hello World Home</h1>
+    </main>
+    // <main className="flex min-h-screen flex-col items-center justify-between p-24">
```

Comment:
You can delete all of this commented out stuff.  Or if you want to keep it for reference, you can delete it in your next PR so it persists in the git history so you can read this PR again for reference in the future

Reply:
Sounds good, I will take it out in next PR